piet-test/src/picture_5.rs

```diff
@@ -21,5 +21,19 @@ pub fn draw(rc: &mut impl RenderContext) -> Result<(), Error> {
     // underline text
     rc.stroke(Line::new((100.0, 52.0), (100.0 + width, 52.0)), &brush, 1.0);
 
+    // get hit test text position and draw a curos
+    // this is for position three, not trailing
+    let hit_test_text_position = layout.hit_test_text_position(3, true);
```

Comment:
The `true` here doesn't match the "not trailing" in the comment. I think it should be set to `false`, which would also have the side effect of making it actually draw a cursor :)

Reply:
the `trailing` arg is set to be removed.